plugins/modules/kibana_alert.py

```diff
@@ -230,17 +229,17 @@ def format_actions(self):
     return formatted_actions
 
   def create_alert(self):
```

Comment:
Why is this function not in the module_utils/kibana.py?

Reply:
Function moved to module_utils/kibana.py